src/content/docs/integrations/kubernetes-integration/installation/install-fargate-integration.mdx

```diff
@@ -0,0 +1,382 @@
+</Callout>
+
+Other components such as `nri-prometheus`, `nri-metadata-injection`, and `nri-kube-events` do not need any extra steps to be deployed on Fargate.
+
+<Callout variant="tip">
+New Relic Logging is not yet supported on Fargate nodes.
+</Callout>
+
+## Installation [#installation]
+
+You can choose between two alternatives for installing New Relic full observability in your EKS Fargate cluster: 
+
+* [Automatic (recommended)](#automatic)
+* [Manual injection](#manual)
+
+Regardless of the approach you choose, the experience is exactly the same after it's installed. The only difference is how the container is injected.
+
+### Automatic injection (recommended) [#automatic]
+
+By default, when Fargate support is enabled, New Relic will deploy an operator to the cluster (`newrelic-infra-operator`). Once deployed, this operator will automatically inject the monitoring sidecar to pods that are scheduled into Fargate nodes, while also managing the creation and the update of `Secrets`, `ClusterRoleBindings`, and any other related resources.
+
+This operator accepts a variety of advanced configuration options that can be used to narrow or widen the scope of the injection. It does this through the use of label selectors for both pods and namespaces.
+
+#### What the operator does [#what-operator-does]
+
+Behind the scenes, the operator sets up a `MutatingWebhookConfiguration`, which allows it to modify the pod objects that are about to be created in the cluster. On this event, and when the pod being created matches the user’s configuration, the operator will:
+
+1. Add a sidecar container to the pod containing the New Relic Kubernetes Integration.
+2. If a secret doesn't exist, create one in the same namespace as the pod containing the New Relic license key, which is needed for the integration to report data.
+3. Add the pod’s service account to a `ClusterRoleBinding` previously created by the operator chart, which will grant this sidecar the required permissions to hit the Kubernetes metrics endpoints.
+
+The `ClusterRoleBinding` grants the following permissions to the pod being injected:
+
+```
+- apiGroups: [""]
+  resources:
+	- "nodes"
+	- "nodes/metrics"
+	- "nodes/stats"
+	- "nodes/proxy"
+	- "pods"
+	- "services"
+  verbs: ["get", "list"]
+- nonResourceURLs: ["/metrics"]
+  verbs: ["get"]
+```
+
+<Callout variant="tip">
+Only pods created after the deployment of the monitoring solution will be injected with the configuration and agent. 
+</Callout>
+
+In order to get the sidecar injected on pods deployed before the operator has been installed, the user needs to manually perform a rollout (restart). New Relic has chosen not to do this automatically in order to prevent unexpected service disruptions and resource usage spikes.
```

Comment:
I would merge these two paragraphs into one.

Reply:
```suggestion
<Callout variant="tip">
In order for the sidecar to be injected on, and therefore to get metrics from pods deployed before the operator has been installed, the user needs to manually perform a rollout (restart) of the affected deployments. This way, when the pods are created, the operator will be able to inject the monitoring sidecar. New Relic has chosen not to do this automatically in order to prevent unexpected service disruptions and resource usage spikes. 
</Callout>
```